Dockerfile

```diff
@@ -24,11 +24,16 @@ RUN wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor
 	chown root:root /etc/apt/sources.list.d/microsoft-prod.list
 
 RUN npm install -g serverless
-RUN apt-get update && apt-get install -y azure-cli dotnet-sdk-3.1 maven azure-functions-core-tools
+RUN npm install -g azure-functions-core-tools@3
+RUN apt-get update
```

Comment:
Can be on the same line as `apt-get install`, best to minimize number of layers in Dockerfile.